docs/AppService/PublishAppService/index.md

```diff
@@ -0,0 +1,81 @@
+## PublishAppService
+
+Type
+: Function
+
+Sender
+: HMI
+
+Purpose
+: Registers a service offered by the HMI on the module
+
+### Request
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|appServiceManifest|[Common.AppServiceManifest](../../common/structs/#)|true||
+
+### Response
+
+#### Parameters
+
+|Name|Type|Mandatory|Additional|
+|:---|:---|:--------|:---------|
+|appServiceRecord|[Common.AppServiceRecord](../../common/structs/#)|false||
+
+### Sequence Diagrams
+|||
+PublishAppService
+![PublishAppService](./assets/PublishAppService.png)
```

Comment:
Typos in `PublishAppService` usages within the diagram. I would also suggest uploading the source files for these diagrams, just in case they need to be edited in the future

Reply:
Resolved in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/139/commits/f2ac175098e32dec1463786a71bc5430fc269ae7